src/ds5/ds5-motion.cpp

```diff
@@ -503,8 +549,15 @@ namespace librealsense
 
     const std::vector<uint8_t> mm_calib_handler::get_fisheye_calib_raw()
     {
-        auto fe_calib_table = (*(ds::check_calib<ds::tm1_eeprom>(_imu_eeprom_raw))).calibration_table.calib_model.fe_calibration;
+        auto fe_calib_table = (*(ds::check_calib<ds::tm1_eeprom>(*_imu_eeprom_raw))).calibration_table.calib_model.fe_calibration;
         uint8_t* fe_calib_ptr = reinterpret_cast<uint8_t*>(&fe_calib_table);
         return std::vector<uint8_t>(fe_calib_ptr, fe_calib_ptr+ ds::fisheye_calibration_table_size);
     }
+
+    bool mm_calib_handler::is_intrinsic_valid()
+    {
+        if ((*_calib_parser) == nullptr) return false;
```

Comment:
hw monitor command is called only the first time, afterwards it's using the flag. however, in the updated PR, this is not to used at all, just keep it for now in case we need it.